document/core/exec/numerics.rst

```diff
@@ -614,14 +614,14 @@ The integer result of predicates -- i.e., :ref:`tests <syntax-testop>` and :ref:
 
 .. _op-iextendn_s:
```

Comment:
change this too?

Reply:
This doesn't actually expand to anything, so I'm fine with keeping it. The other made it difficult to understand the def from the Latex source, because the M on the rhs seemed to becoming out of nowhere.